CMakeLists.txt

```diff
@@ -347,8 +347,8 @@ if (SOLARIS)
 endif ()
 
 if (CMAKE_COMPILER_IS_GNUCC)
-  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.8")
-    message(FATAL_ERROR "ArangoDB requires GCC 4.9 or newer.")
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "5.3")
+    message(FATAL_ERROR "ArangoDB requires GCC 5.4 or newer.")
   elseif(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "6.9.9")
```

Comment:
@dothebart does this mean 5.3 is allowed or not (VERSION_LESS is "less" or is it "less or equal")?